### PR TITLE
Create RealBattery.netkan

### PR DESCRIPTION
**Pull request by blackliner**
a new mod made by me. Thanks!

### CODE REVIEW

File: NetKAN/RealBattery.netkan [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "RealBattery",
+    "$kref"        : "#/ckan/github/blackliner/RealBattery",
+    "license"      : "MIT",
+    "ksp_version"  : "1.2",
```
Comment (politas) [inline NetKAN/RealBattery.netkan]: You have an AVC .version file, so you could replace this line with the following and KSP compatibility on CKAN will track with AVC:

    "$vref"        : "#ckan/ksp-avc",

Comment (blackliner) [inline NetKAN/RealBattery.netkan]: thanks for the hint. I knew that exists, but i was not shure if it works 100%. Ill try it :-)

**Comment by blackliner** [comment]
ok, what went wrong now?

**Comment by dbent** [comment]
@blackliner 

```
191 [1] FATAL CKAN.NetKAN.Program (null) - Could not parse reference: "#ckan/ksp-avc"
```

The `$vref` property should be `#/ckan/ksp-avc` (Note the extra slash at the beginning).

**Comment by politas** [comment]
Sorry, that's my bad.

**Comment by politas** [comment]
Looks all good now. Merging with thanks!

**Pull request merged**